src/Symfony/Component/Validator/Tests/Constraints/UrlValidatorTest.php

```diff
@@ -144,6 +168,46 @@ public function testInvalidUrls($url)
             ->assertRaised();
     }
 
+    /**
+     * @dataProvider getInvalidRelativeUrls
+     * @dataProvider getInvalidUrls
+     */
+    public function testInvalidRelativeUrl($url)
+    {
+        $constraint = new Url(array(
+            'message' => 'myMessage',
+            'relativeProtocol' => true,
+        ));
+
+        $this->validator->validate($url, $constraint);
+
+        $this->buildViolation('myMessage')
+            ->setParameter('{{ value }}', '"'.$url.'"')
+            ->setCode(Url::INVALID_URL_ERROR)
+            ->assertRaised();
+    }
+
+    public function getInvalidRelativeUrls()
+    {
+        return array(
```

Comment:
Any reason why we are not using `[]` instead of `array`

Reply:
It's because [Symfony coding standards](https://symfony.com/doc/current/contributing/code/standards.html). We can't easily change the existing code from `array()` to `[]` ... so using `[]` only for the new code would be consistent and that's why we use `array()` everywhere.

Reply:
This started as a patch for 2.8 so I needed to use the `array` keyword to not break code in PHP 5.3 seeing this is supported with Symfony 2.x. I definitely prefer the brackets and could changes it.